src/applications/mhv/medications/sass/medications.scss

```diff
@@ -172,6 +179,10 @@
       }
     }
   }
+  @media (max-width: $small-screen) {
+    padding-left: 0px;
+    padding-right: 0px;
+  }
```

Comment:
this would apply to all the pages I believe? It looks like this is also used in the Routes.jsx file surrounding all the components.

Reply:
no this one is to remove the padding created by the .vads-l-grid-container on the landing page, its located under the .landing page {} class selector.